### ISSUE TITLE
pythonroom-style error location highlighting

### ISSUE THREAD

**Issue by willwhitney** [willwhitney/hydrogen#87]

**Comment by rgbkrk** [comment]
We definitely need a better model for tracebacks in Jupyter. Right now, as you know, it's a strange split of error names and values.

Things we know in the frontend:

* The code we sent over
* The traceback we get in response

/cc @smashwilson

**Comment by smashwilson** [comment]
Yeah, it might be possible to generate an event that [stacktrace](https://github.com/smashwilson/stacktrace) could consume. This may require work on the stacktrace end though, considering that I don't think I've worked on it since the package subscription spec came out.

**Comment by smashwilson** [comment]
Oh no, I [totally did](https://github.com/smashwilson/stacktrace/blob/master/package.json#L26)! Well look at that.
